ingesters/http.md

```diff
@@ -370,6 +374,19 @@ Multiple Tag-Match pairs can be given:
 Tag-Match=foo:bar ping:pong
 ```
 
+## Amazon Firehose support
+
+The `Amazon-Firehose-Listener` type supports the [Amazon Firehose](https://aws.amazon.com/firehose/) API. The Firehose API requires the use of HTTPS and an authentication token. 
+
+```
+[Amazon-Firehose-Listener "foo"]
+	URL="/foo"
+	TokenValue="thisisyourtoken" #set the access control token
+	Tag-Name=bar
+```
+
+In the above example, the HTTP ingester will listen on the `/foo` path for an Amazon Firehose request, authenticated with the token "thisisyourtoken", and ingesting to tag "bar". In the AWS console, you would set the Firehose endpoint to `your.domain/foo`, and provide the same token.
```

Comment:
Please add the config parameter table here too, verbose is good

- URL
- TokenValue
- Tag-Name
- Ignore-Timestamps
- Preprocessor set